doc/specification.md

```diff
@@ -77,6 +77,12 @@ Otherwise, the device discards the `AFTOperation` and returns a `ModifyResponse`
 
 `ModifyRequest.election_id` MUST be non zero. When a device receives an value of 0, it should close the `Modify` RPC and set `Status.code` to `INVALID_ARGUMENT`. The election ID can only be increased monotonically by a client during a RPC session. This simplifies server implementation.
 
+#### 4.1.2.1 Election ID Reset
+
+There is no motivation to provide anyway for clients to reset the election ID on the device. If clients lose track of the highest election ID known by the device, clients can learn the value via `ModifyResponse.election_id` from the device (see x.y.z for more details).
+
+It is possible that in some scenarios (e.g., daemon crash, device reboot) the device might lose the highest learned election ID and hence set the value to 0. However, a device SHOULD NOT promptly reset the value to 0 in any cases(e.g., all clients disconnect). This helps reduce the chaince of non-primary client programming the device in some failure scenarios (e.g., some error happens on clients side that might lead to split-brain among clients and also cause all clients disconnect and then reconnect).
```

Comment:
Is the value set to zero -- or the value is reset (becomes unset)? Since we said 0 is an invalid value -- it seems like we might want to consider our language of saying what "set to zero" means vs. "is unset". Equally, if a server has just started, and has no clients connected, it might subsequently operate in `ALL_PRIMARY` mode, which would mean that setting the election ID to any value would be invalid.

Reply:
This is a very good point. I should have paid more attention to the diff between `reset` and `unset`. Reworded this part. PTAL. Thanks.